guides/common/modules/proc_creating-custom-snippets-to-apply-errata.adoc

```diff
@@ -0,0 +1,36 @@
+[id="creating-custom-snippets-to-apply-errata_{context}"]
+= Creating custom snippets to apply errata
+
+Custom snippets allow you to execute custom code when applying errata on hosts.
```

Comment:
I recommend a less anthropomorphic style. Perhaps, "You can use custom snippets to execute custom code when applying errata on hosts."

Reply:
I am not sure about the discrepancy between this line and the title. Do custom snippets apply errata or do they run custom code?

Reply:
Applied. The custom snippets can run arbitrary code. They generally prepare a host to have errata applied or perform a cleanup.